docs/reference/config.md

```diff
@@ -136,6 +136,15 @@ commitAuthor:
 commitMessage: "chore(release): this is a custom release message"
 ```
 
+## DirtyFiles
+```{ .yaml .annotate linenums="1" }
+# Allowing specific case sensitive files to allow using uplift to
+# add unedited but changed files as part of your CI
+
+dirtyFiles:
+    - /path/to/file
+```
```

Comment:
I would prefer this to be included within the existing `git` section of the uplift config. I think `dirtyFiles` is a good enough description for its purpose. I would be tempted to update the description to something along the lines:

```yaml
git:
  # A list of files that are changed outside of uplift but still require uplift to manage
  # to manage them. Any modified file not detected in this list will prevent uplift
  # from running due to the repository being in a dirty state
  dirtyFiles:
    - file.txt
    - path/to/file.txt
```

Reply:
You will also need to update the schema to include this new field and its description.